test/e2e/search_test.go

```diff
@@ -165,8 +166,16 @@ registries = ['{{.Host}}:{{.Port}}']`
 	})
 
 	It("podman search v2 registry with empty query", func() {
-		search := podmanTest.Podman([]string{"search", "registry.fedoraproject.org/"})
-		search.WaitWithDefaultTimeout()
+		var search
```

Comment:
Compiler is yelling because it's not sure what type this is... I'm honestly not either. You might need to make this `search := podmanTest.Podman(...)` so the compiler infers the type

Reply:
>I'm honestly not either. 

Oh, **thank you**. I've been pulling my hair out trying to figure out the type (I can't get ginkgo working on my laptop, so I can't experiment either). 

If I use `:=` won't it fall out of scope outside the `for` loop? /me doesn't grok golang.

Reply:
var search *PodmanSessionIntegration

Reply:
should work

Reply:
Yeah, `:=` is just sugar for `var $NAME <inferred type> = ...`

Reply:
@mheon, I know that much^Wlittle about Go; my question was, doesn't `:=` declare within curlybrace scope? If I did `session := ...`, could I use `session` in the two `Expect()`s below?

Reply:
Yes - it'll leak into the sub-scope created by the curly braces (though you can declare another `session` within that sub-scope to shadow it in there).

Reply:
I'm still not using the right words. Let me try this way. Will this work? I wouldn't expect it to.
```go
    for .... {
        myvar := autodeclared
        ....
    }
    if myvar ...     // <<---- referencing myvar outside of the above "for" scope
```

Reply:
PS thank you @rhatdan, that seems to be getting it much further along.